src/commands/is-disclosed.ts

```diff
@@ -0,0 +1,67 @@
+import { BaseCommand } from "../baseCommand";
+import { Formater, OUTPUT_FORMAT } from "../formater";
+import { confirmedFlag, hiddenPropertyKeyFlag, unikidFlag } from "../utils";
+import { GetPropertyValueCommand } from "./get-property-value";
+
+export class IsDisclosedCommand extends GetPropertyValueCommand {
+    public static description = "Check if UNIK has disclosed one or more explicit values.";
+
+    public static examples = [
+        `$ uns is-disclosed --unikid 636795fff13c8f2d2fd90f9aa124d7f583920fce83588895c917927ee522db3b --network devnet`,
+    ];
+
+    public static flags = {
+        ...GetPropertyValueCommand.flags,
+        ...confirmedFlag,
+        ...unikidFlag("The UNIK token"),
+        ...hiddenPropertyKeyFlag("explicitValue"),
+    };
+
+    protected getAvailableFormats(): Formater[] {
+        return [OUTPUT_FORMAT.json, OUTPUT_FORMAT.yaml, OUTPUT_FORMAT.raw];
+    }
+
+    protected getCommand(): typeof BaseCommand {
+        return IsDisclosedCommand;
+    }
+
+    protected async do(flags: Record<string, any>): Promise<any> {
+        try {
+            const res = await super.do(flags);
```

Comment:
And if you don't extend GetPropertyValueCommand, you can only call getProperty function with "explicitValue" property, or didResolve passedExplicitValue?explicitValue